spec/javascripts/utilities/useClickOutside.spec.tsx

```diff
@@ -12,15 +12,15 @@ const HookedComponent: FunctionComponent<{ callback: jest.Mock }> = ({ callback
   return <div ref={ref}>I am hooked</div>
 }
 
-it('should and and remove a mousedown event listener', () => {
+it('should add and remove a mousedown event listener', () => {
   const addEventListener = jest.fn()
   const removeEventListener = jest.fn()
 
   document.addEventListener = addEventListener
   document.removeEventListener = removeEventListener
 
   const wrapper = mount(<HookedComponent callback={jest.fn()} />)
-  expect(addEventListener).toHaveBeenCalledTimes(1)
+  expect(addEventListener).toHaveBeenCalledWith('mousedown', expect.any(Function))
```

Comment:
Not checking the number of calls?

Reply:
The important thing is to check the `mousedown` event has been registered, since there are other calls which might not be related with the useClickOutside hook. Hence the change.

Reply:
But it's useful to check the mouse event has been registered once, right?

Reply:
There's not an specific way of filtering those calls, and though we've checked it manually, we consider it isn't worth the fuzz.